pkg/hoist/hoist_launchable.go

```diff
@@ -98,7 +112,7 @@ func (hl *Launchable) enable() (string, error) {
 
 	// providing an enable script is optional, ignore those errors
 	if err != nil && !os.IsNotExist(err) {
-		return output, util.Errorf("Could not enable %s: %s", hl.Id, err)
```

Comment:
launchable ID will always be logged at a higher layer. I wanted to avoid redundant log messages like

```json
{"err":"Could not enable foo: exit status 1","launchable":"foo","message":"Enable failed"}
```

I'll see what I can do about the stack trace but I think in this case it's not especially meaningful. An EnableError is obviously coming from the Enable function, and with the type information, all of our logging around enable/disable should be easier to trace.

Philosophical tangent: I think stack traces in error messages are a bandage for errors being too aggressively stringified or lacking in context. If this was an exception-based language then obviously they'd be unavoidable, but more effective structuring of error values should make stack traces in errors mostly obsolete, especially in short functions like this one. End tangent.